alerter.py

```diff
@@ -16,10 +19,11 @@ def alert_in_celcius(farenheit):
         # However, this code doesn't count failures!
         # Add a test below to catch this bug. Alter the stub above, if needed.
         global alert_failure_count
-        alert_failure_count += 0
+        alert_failure_count += 1
 
 
 alert_in_celcius(400.5)
 alert_in_celcius(303.6)
+assert(alert_failure_count == 0)

```

Comment:
the tests should include all input values ex. below threshold, above threshold and threshold. Then the failure count should be asserted. Here the assert value is zero! The test cases should be effective in order to express the expectation from the production code.